modules/admin/src/resources/scss/components/_luya.scss

```diff
@@ -52,7 +52,7 @@
 }
 
 .luya-subnav {
-    background-color: map-get($grays, 100);
+    background-color: map-get($grays, 200);
```

Comment:
@dev7ch now the luyasubnav appears grey again, this should not be gray and should have the same color as the container to the right